src/index.ts

```diff
@@ -100,20 +93,17 @@ export async function typescriptOfSchema(
     importTs = `import {${symbols}} from "${options.jsonTypesFile}";\n\n`;
   }
 
-  const tableNames = tables.map(t =>
-    normalizeName(getTableTypeName(t, schema, !!options.prefixWithSchemaNames)),
```

Comment:
Previously the naming logic was duplicated across `typescript.ts` and `index.ts`. Now it's all done in `typescript.ts`.